Makefile

```diff
@@ -95,6 +95,14 @@ E2E_TEST_FLAGS = "--skip-deletion=false" -timeout 25m # See README.md, default g
 # see target "image-build"
 IMAGE_BUILD_FLAGS ?= --build-arg USE_LOCAL=false
 
+# Prometheus-Unit Tests Parameters
+PROMETHEUS_CONFIG_YAML ?= $(shell pwd)/config/monitoring/prometheus/apps/prometheus-configs.yaml
```

Comment:
gnu make has CURDIR variable, no point to call shell probably. Or you have something special here?